src/view-port-cache.js

```diff
@@ -0,0 +1,38 @@
+import {transient} from 'aurelia-dependency-injection';
+
+@transient()
+export class ViewPortCache {
+  controllers = Object.create(null);
+
+  get(viewPortName, viewPortConfig, navigationInstruction) {
```

Comment:
Should `viewPortConfig` be the first argument, as the others can can be ommited if `viewPortConfig.cacheable` is `false`.